staging.datastage.io/manifest.json

```diff
@@ -38,7 +38,7 @@
       "action": "export",
       "container": {
         "name": "job-task",
-        "image": "quay.io/cdis/pelican:0.1.9",
+        "image": "quay.io/cdis/pelican:0.2.0",
```

Comment:
It should be `0.2.1`.

Reply:
0.2.1 isn't in quay.

Reply:
@emalinowski I have released a new version for `pelican`: `0.2.2`.

And use `pelican-export` image: https://quay.io/repository/cdis/pelican-export?tab=tags